addons/im_livechat/static/src/core/web/livechat_channel_info_list.js

```diff
@@ -0,0 +1,22 @@
+import { ActionPanel } from "@mail/discuss/core/common/action_panel";
+import { Component } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { createDocumentFragmentFromContent } from "@mail/utils/common/html";
+
+export class LivechatChannelInfoList extends Component {
+    static components = { ActionPanel };
+    static template = "im_livechat.channelInfoList";
+    static props = ["thread"];
+
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+    }
+
+    onBlurNote() {
```

Comment:
Ok to merge like that, but in terms of UI it's really not clear when the save is done or not.